.build/tar/build_exporters.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# The use of >/dev/null is to supress stdout, but allow stderr to come thorugh
+# This is because installing the tools and building makes a lot of output
+
+echo "Installing tools needed"
+
+# Install git and maven for building the exporters
+sudo apt-get update >/dev/null
+sudo apt-get install -y git >/dev/null
+sudo apt-get install -y maven >/dev/null
```

Comment:
This in particular seems a weird thing to supress

Reply:
I'll let that come through